variables.yml

```diff
@@ -1,7 +1,7 @@
 networks:
   development:
-    build_tag: v0.40.0
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.40.0-3200-latest
+    build_tag: v0.40.1
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.40.1-3200-ae11
```

Comment:
we've been using latest, can we use that instead?

Reply:
[v0.40.1-3200-latest](https://hub.docker.com/layers/moonbeamfoundation/moonbeam-tracing/v0.40.1-3200-latest/images/sha256-f69b36f53b9b173fca333aa1d822101761d9bff6275aa1cd6af03fd96d62a036?context=explore)

Reply:
I prefer using the official tag being used by the release comms

Reply:
Haha, we change our mind on this like every 6 months 😂